SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -121,6 +124,13 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	UPROPERTY()
 	USnapshotManager* SnapshotManager;
 
+	// Setting ActorReplicationRateLimit in DefaultEngine.ini will limit the number of actors which are replicated per tick to the number specified.
```

Comment:
Setting ActorReplicationRateLimit in the `/Script/SpatialNetDriver` section, or whatever the proper section name is.